wp-cache-sanitizer.php

```diff
@@ -0,0 +1,726 @@
+  // wp-cache-config.php file as either true, false, 0 or 1
+  const ALLOWED_SETTING_BOOLEANS = array(
+      "cache_compression",
+      "cache_enabled",
+      "cache_gc_email_me",
+      "cache_jetpack",
+      "cache_rebuild_files",
+      "dismiss_gc_warning",
+      "dismiss_htaccess_warning",
+      "dismiss_readable_warning",
+      "ossdlcdn",
+      "super_cache_enabled",
+      "use_flock",
+      "wp_cache_anon_only",
+      "wp_cache_clear_on_post_edit",
+      "wp_cache_cron_check",
+      "wp_cache_debug_email",
+      "wp_cache_debug_level",
+      "wp_cache_debug_to_file",
+      "wp_cache_disable_utf8",
+      "wp_cache_front_page_checks",
+      "wp_cache_hello_world",
+      "wp_cache_hide_donation",
+      "wp_cache_make_known_anon",
+      "wp_cache_mfunc_enabled",
+      "wp_cache_mobile",
+      "wp_cache_mobile_enabled",
+      "wp_cache_mobile_whitelist",
+      "wp_cache_mod_rewrite",
+      "wp_cache_mutex_disabled",
+      "wp_cache_no_cache_for_get",
+      "wp_cache_not_logged_in",
+      "wp_cache_object_cache",
+      "wp_cache_preload_email_me",
+      "wp_cache_preload_on",
+      "wp_cache_preload_taxonomies",
+      "wp_cache_refresh_single_only",
+      "wp_cache_shutdown_gc",
+      "wp_cache_slash_check",
+      "wp_super_cache_advanced_debug",
+      "wp_super_cache_comments",
+      "wp_super_cache_debug",
+      "wp_super_cache_front_page_check",
+      "wp_super_cache_front_page_clear",
+      "wp_super_cache_front_page_notification",
+      "wp_super_cache_late_init",
+      "wp_supercache_304",
+      "wp_supercache_cache_list",
+  );
+
+  // The list of allowed global integer variables that are written to the wp-cache-config.php
+  const ALLOWED_SETTING_INTS = array(
+      "cache_badbehaviour",
+      "cache_awaitingmoderation",
+      "cache_domain_mapping",
+      "cache_time_interval",
+      "cache_max_time",
+      "cache_rejected_uri",
+      "wp_cache_preload_interval",
+      "cache_wptouch",
+      "sem_id",
+  );
+
+  // The list of allowed global string variables that are written to the wp-cache-config.php
+  const ALLOWED_SETTING_STRINGS = array(
+      "cache_acceptable_files",
+      "cache_badbehaviour_file",
+      "cache_no_adverts_for_friends",
+      "cache_page_secret",
+      "cache_rejected_user_agent",
+      "cache_scheduled_time",
+      "cache_schedule_interval",
+      "cache_schedule_type",
+      "cached_direct_pages",
+      "file_prefix",
+      "wp_cache_debug_ip",
+      "wp_cache_debug_log",
+      "wp_cache_home_path",
+      "wp_cache_mobile_browsers",
+      "wp_cache_mobile_groups",
+      "wp_cache_mobile_prefixes",
+      "wp_cache_pages",
+      'wp_cache_plugins_dir',
+      "wp_cache_preload_email_volume",
+      "wp_cache_preload_posts",
+      "wp_super_cache_front_page_text",
+      "wptouch_exclude_ua",
+  );
+
+  // The default list of mobile browsers taken from wp-cache.php:648
+  const MOBILE_BROWSERS = array(
```

Comment:
We can, although we'll likely wait until after the exporter has been integrated to do so.